sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -93,6 +97,23 @@
       <version>1.3.5</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
       <scope>test</scope>
     </dependency>
+
+    <!-- graalvm test -->
+    <dependency>
+      <groupId>org.graalvm.buildtools</groupId>
+      <artifactId>junit-platform-native</artifactId>
+      <version>0.9.2</version> <!-- {x-version-update;org.graalvm.buildtools:junit-platform-native;external_dependency} -->
+      <scope>test</scope>
+    </dependency>
+
+    <!-- all graalvm config is in core-experimental -->
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-experimental</artifactId>
+      <version>1.0.0-beta.17</version> <!-- {x-version-update;unreleased_com.azure:azure-core-experimental;dependency} -->
+      <scope>test</scope>
+    </dependency>
```

Comment:
I'm not fully clear on what this is for?

Reply:
This is to run native tests for appconfig. Since all graalvm config is in core-experimental, I had to include this as a test dependency.

Reply:
Got it. This will go away soon when it moves into core?

Reply:
Yes, this will not be required once we move all the graalvm config from experimental to core.